### PR TITLE
Update README with recent changes and support for Skaffold v2

### PR DESCRIPTION
**Pull request by pstranak-sw**

### CODE REVIEW

File: skaffold.yaml [hunk 1]
```diff
@@ -3,40 +3,38 @@ kind: Config
...
-    - image: swi-k8s-opentelemetry-collector
-      docker:
-        dockerfile: build/docker/Dockerfile
 deploy:
   helm:
     releases:
-      - name: monitoring
-        remoteChart: prometheus
-        imageStrategy:
-          fqn: {}
```
Comment (pstranak-sw) [inline skaffold.yaml:18]: This is the default value, so it's not necessary.

**Review by juraj-paska — DISMISSED** [review]

File: skaffold2.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      namespace: monitoring
+      createNamespace: true
+      repo: https://prometheus-community.github.io/helm-charts
+  kustomize:
+    paths:
+    - tests/deploy/base
+    buildArgs:
+    - --load-restrictor LoadRestrictionsNone
+deploy:
+  kubectl: {}
```
Comment (pstranak-sw) [inline skaffold2.yaml:26]: When using both `helm` and `kustomize` manifests, we have to explicitly declare using `kubectl` and `helm` as the deployers. `kubectl` is used automatically only if there is no other deployer defined.

**Review by gantrior — APPROVED** [review]
Looks good

**Comment by pstranak-sw** [comment]
Rebased on https://github.com/solarwinds/swi-k8s-opentelemetry-collector/pull/105 to fix the build checks.

**Review by juraj-paska — APPROVED** [review]

**Pull request merged**